org.kde.trojita.yml

```diff
@@ -0,0 +1,41 @@
+app-id: org.kde.trojita
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+base: io.qt.qtwebkit.BaseApp
+base-version: "5.15-21.08"
+command: trojita
+rename-desktop-file: trojita.desktop
+rename-icon: trojita
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --persist=.abook
+
+modules:
+  - name: trojita
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DWITH_TESTS=OFF
+      - -DWITH_QT5=ON
+      - -DWITH_ZLIB=ON
+      - -DWITH_RAGEL=OFF
+    post-install:
+      - rm ${FLATPAK_DEST}/share/appdata/trojita.appdata.xml
+      - install org.kde.trojita.appdata.xml ${FLATPAK_DEST}/share/appdata/org.kde.trojita.appdata.xml
```

Comment:
you should use the patch you submitted instead.

Reply:
File names differ. The patch will be messy.